src/evergreen_names.rs

```diff
@@ -93,8 +93,13 @@ pub const MULTIVERSION_EXCLUDE_TAGS: &str = "multiversion_exclude_tags_version";
 // Build Variant expansions.
 /// Name of large distro for build variant.
 pub const LARGE_DISTRO_EXPANSION: &str = "large_distro_name";
-/// List of build variant names delimited by spaces to generate burn_in_tags for.
-pub const BURN_IN_TAG_BUILD_VARIANTS: &str = "burn_in_tag_buildvariants";
+/// List of build variant names delimited by spaces to generate burn_in_tests for.
+pub const BURN_IN_TAG_INCLUDE_BUILD_VARIANTS: &str = "burn_in_tag_include_build_variants";
+/// Generate burn_in_tests for all required and suggested build variants.
+pub const BURN_IN_TAG_REQUIRED_AND_SUGGESTED_BUILD_VARIANTS: &str =
+    "burn_in_tag_required_and_suggested_build_variants";
```

Comment:
It took a while for me to understand that this expansion should hold a boolean value. Can we rename it to `burn_in_tag_include_all_required_and_suggested_build_variants`? I think this way it will be more clear.

Reply:
Yeah, makes sense. Updated.